<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paint</class>
 <widget class="QWidget" name="paint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>1240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1040</width>
    <height>840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>10</y>
     <width>191</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>基站1坐标</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="anchor1_pos_x"/>
    </item>
    <item>
     <widget class="QLineEdit" name="anchor1_pos_y">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="anchor1_pos_z"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>基站2坐标</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="anchor2_pos_x"/>
    </item>
    <item>
     <widget class="QLineEdit" name="anchor2_pos_y">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="anchor2_pos_z"/>
    </item>
    <item>
     <widget class="QPushButton" name="anchor_pos_button">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>输入</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>820</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="Anchor_2">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>二基站</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Anchor_3">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>三基站</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="Anchor_4">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>四基站</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Startpaint_button">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>790</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始绘制</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="track_box">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>840</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>是否显示轨迹</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tmp_textEdit">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>330</y>
     <width>311</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
